src/xrpld/rpc/detail/RPCHelpers.cpp

```diff
@@ -385,20 +385,23 @@ ledgerFromRequest(T& ledger, JsonContext& context)
         return getLedger(ledger, ledgerHash, context);
     }
 
-    auto const index = indexValue.asString();
+    if (indexValue.isConvertibleTo(Json::stringValue))
+    {
+        auto const index = indexValue.asString();
 
-    if (index == "current" || index.empty())
-        return getLedger(ledger, LedgerShortcut::CURRENT, context);
+        if (index == "current" || index.empty())
+            return getLedger(ledger, LedgerShortcut::CURRENT, context);
 
-    if (index == "validated")
-        return getLedger(ledger, LedgerShortcut::VALIDATED, context);
+        if (index == "validated")
+            return getLedger(ledger, LedgerShortcut::VALIDATED, context);
 
-    if (index == "closed")
-        return getLedger(ledger, LedgerShortcut::CLOSED, context);
+        if (index == "closed")
+            return getLedger(ledger, LedgerShortcut::CLOSED, context);
 
-    std::uint32_t iVal;
-    if (beast::lexicalCastChecked(iVal, index))
-        return getLedger(ledger, iVal, context);
+        std::uint32_t iVal;
```

Comment:
Please avoid Hungarian

Reply:
This line wasn't touched by me, I don't see the point in increasing the diff to change a minor variable name